src/ext.js

```diff
@@ -26,7 +26,23 @@ const toggleOptions = [
   },
 ];
 
+const fontSizeOptions = [
+  {
+    value: 'responsive',
+    translation: 'properties.responsive',
+  },
+  {
+    value: 'relative',
```

Comment:
What is the difference between these?

Reply:
Responsive will always fit the text inside the button, so it considers the length of the label. 
Relative used the box size of the button to calculate the font size. It is independent of the label (length). This means that buttons with the same size will always have the same font size
Fixed is a fixed size in pixels

Reply:
So what is the relative scale relative to? Seems like you are reusing the same slider, that goes from 0.2 to 1. it just seem very arbitary to me. I see that this is the simplest solution. Use the same slider, adjust the values so a "normal" label fits in a good way for those values

Reply:
Hey @haxxmaxx 👋 

I need to continue Niek's work on this one and i'm a bit confused with this comment of yours. what do you mean by this: "adjust the values so a "normal" label fits in a good way for those values"? 

The value is relative to **the box size** of the button, but do you mean that **the scale factor** here (numbers from 0.2 to 1) is seems a bit odd to you? 

are you concerned with the numbers that does not means something to the end user (I mean like if we stand in shoes of an end user (your concern would be) if it is relative -> then what 0.2 and 1 means? like relative to what? and what defines 0.2 and 1? )

am i understanding your question correctly? 🙂
if so, what would be the correct behaviour here and your suggestion, i would need some more info to fix it and get it going.

Reply:
we decided to get this one in, with out the Fluid options until we get more clarification on it, here is the extended work related to this PR: https://github.com/qlik-oss/sn-action-button/pull/331

Thanks @niekvanstaveren  for this 🙏